bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -38,18 +40,21 @@
 os.environ['BENTOML_HOME'] = '/tmp/bentoml/'
 from bentoml import load  # noqa
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
I think we can remove this and hide debug logs by default, those lots are usually only used for BentoML developer/contributor to debug the feature and not so much for BentoML user.  For BentoML developer, we can always enable debug log with a simple environment variable change `BENTOML__LOGGING__LOGGING_LEVEL=debug`

Reply:
What is the default logging level for BentoML? Is it info? When I don't explicitly set the logging level to be DEBUG or INFO, my logger.info() statements don't show up

Reply:
The default is INFO https://github.com/bentoml/BentoML/blob/master/bentoml/configuration/default_bentoml.cfg#L30

I just realized this will not be using the logger set for BentoML module because `lambda_app.py` will be in a separate directory. To use BentoML's logging config here, you will need to use `logger = logging.getLogger('bentoml.lambda_app')`